frontend/src/app/services/model/model.service.ts

```diff
@@ -28,8 +28,8 @@ export interface Model {
   tool: Tool;
   version: ToolVersion | null;
   type: ToolType | null;
-  t4c_model_id: number | null;
-  git_model_id: number | null;
+  t4c_models: Object | null;
+  git_models: Object | null;
```

Comment:
You should specify the type of `t4c_models` and `git_models`, and also rather use the interrogative syntax.

```typescript
version?: ToolVersion;
type?: ToolType;
t4c_models?: T4CModel[];
git_models?: GitModel[];
```

Reply:
Type is probably indeed. Thanks for making me aware.

Reply:
You should anyways rebase on v2/staging, because the `t4c_models` and `git_models` properties are handled anyways tho.